openedx/features/course_duration_limits/access.py

```diff
@@ -109,7 +109,7 @@ def get_user_course_expiration_date(user, course):
         # Content availability date is equivalent to max(enrollment date, course start date)
         # for most people. Using the schedule date will provide flexibility to deal with
         # more complex business rules in the future.
-        content_availability_date = enrollment.schedule.start
+        content_availability_date = enrollment.schedule.start_date
```

Comment:
@hunytalk does this alter the production logic to use this new column?  Can we split this into a separate PR to run before removing the column?

Reply:
Also, does this mean the old column start is currently still in use in production, after we already started removing writes to it in the previous PR???

Reply:
Note for reviewers: we extracted this and a few other changes into a separate PR: https://github.com/edx/edx-platform/pull/22975